File: kafka-cassandra-connect/CassandraTables.js

module.exports.InsertToDevices =  async (client, traffic1, num1, traffic2, num2) => {

	const query = 'INSERT INTO devices (id, traffic1, num1, traffic2, num2) VALUES (?, ?, ?, ?, ?)';
	let timestamp = String(Date.now())
	const params = [timestamp, traffic1, num1, traffic2, num2];

	client.execute(query,params, { prepare: true })
	.then((res) => {
		console.log("record inserted successfully")
	})

}

File: kafka_cassandra_connect/cassandra_tables.py
import time


def insert_to_devices(session, traffic1, num1, traffic2, num2):

    timestamp = int(time.time())
    session.execute(
    """
    INSERT INTO devices (id, traffic1, num1, traffic2, num2)
    VALUES (%s, %s, %s, %s, %s)
    """,
    (timestamp, traffic1, num1, traffic2, num2)
    )
    print("record added succefully to Cassandra")